zoo-css-challenge/index.html

```diff
@@ -30,40 +32,42 @@ <h1>Birmingham Zoo</h1>
 		<section class="bears">
 			<h2>Bears</h2>
 			<p>Did you know that a bear's closest relative is the seal? Or that bears can smell over 2,000 times better than humans?</p>
-			<div class="image-container">
+			<div class="image-container" style="display:flex; justify-content:center">
```

Comment:
```suggestion
			<div class="image-container">
```
We should add this CSS in the `.image-container` rule in the style.css file instead